rotkehlchen/chain/ethereum/modules/makerdao/decoder.py

```diff
@@ -436,6 +443,68 @@ def decode_cdp_manager_events(self, context: DecoderContext) -> DecodingOutput:
 
         return DEFAULT_DECODING_OUTPUT
 
+    def decode_sdai_events(self, context: DecoderContext) -> DecodingOutput:
+        if context.tx_log.topics[0] == SDAI_DEPOSIT:
+            # DAI -> SDAI: owner receives sdai
+            owner_address = self._get_address_or_proxy(hex_or_bytes_to_address(context.tx_log.topics[2]))  # noqa: E501
+            to_address = owner_address
+            from_address: Optional[ChecksumEvmAddress] = self.sdai.evm_address
+        elif context.tx_log.topics[0] == SDAI_REDEEM:
+            # SDAI -> DAI: owner deposits sdai
+            owner_address = self._get_address_or_proxy(hex_or_bytes_to_address(context.tx_log.topics[3]))  # noqa: E501
+            from_address = owner_address
+            to_address = self.sdai.evm_address
```

Comment:
if possible comments shouldn't add lines of code

```suggestion
        if context.tx_log.topics[0] == SDAI_DEPOSIT:   # DAI -> SDAI: owner receives sdai
            owner_address = self._get_address_or_proxy(hex_or_bytes_to_address(context.tx_log.topics[2]))  # noqa: E501
            to_address = owner_address
            from_address: Optional[ChecksumEvmAddress] = self.sdai.evm_address
        elif context.tx_log.topics[0] == SDAI_REDEEM:  # SDAI -> DAI: owner deposits sdai
            owner_address = self._get_address_or_proxy(hex_or_bytes_to_address(context.tx_log.topics[3]))  # noqa: E501
            from_address = owner_address
            to_address = self.sdai.evm_address
```

Reply:
+1